salt/output/json_out.py

```diff
@@ -54,7 +54,7 @@ def output(data):
     '''
     try:
         if 'output_indent' not in __opts__:
-            return json.dumps(data, default=repr, indent=4)
+            return json.dumps(data, default=repr, indent=4, encoding='latin1')
```

Comment:
The utf-16 encoding is also breaking the command. 
I dont know if it makes sense to run the python command on the minion as the encodeing happens on the server (as far as I understand)...